app/Traits/ArclightConnection.php

```diff
@@ -25,9 +25,16 @@ private function fetchArclight($path, $language_id = null, $include_refs = false
         $path .= '&'.$parameters;
 
         try {
-            $results = json_decode(file_get_contents($path));
+            $ctx = stream_context_create(array('http'=>
+                array('timeout' => 5)
+            ));       
+            $results = json_decode(file_get_contents($path, false, $ctx));
```

Comment:
just to confirm, we are not retrieving media content, correct?  We should only be retrieving the CDN references which then are returned to the browser so they can access the content on their network connection